### ISSUE TITLE
AWS: Security group not found

### ISSUE THREAD

**Issue by abhinav-chittora** [GoogleCloudPlatform/terraformer#164]
When applying ec2_instance which was created by **terraformer import aws --resources=sg,ec2_instances,subnet,vpc --regions=us-east-1**, I am getting error that the security group is not available, however, in AWS console, I can see the security group. 

Environment details: - 
**Terraformer Version:**
````
terraformer version
Terraformer v0.7.6
````

**Terraform Version:**
````terraform version
Terraform v0.12.6
+ provider.aws v2.22.0````

**Comment by sergeylanzman** [comment]
it's VPC security groups?

**Comment by abhinav-chittora** [comment]
@sergeylanzman yes, it is VPC Security groups.

**Comment by sergeylanzman** [comment]
Can you show me you error and output?

**Comment by abhinav-chittora** [comment]
```
GGN-136091-C02X619VJHD5:us-east-1 ageek$ terraform plan

Error: Unsupported attribute

  on instance.tf line 30, in resource "aws_instance" "i-01a3c8837f5c1a2ab_Rahul_UF3":
  30:   subnet_id         = "${data.terraform_remote_state.subnet.aws_subnet_subnet-5e33fa70_id}"

This object has no argument, nested block, or exported attribute named
"aws_subnet_subnet-5e33fa70_id".


Error: Unsupported attribute

  on instance.tf line 38, in resource "aws_instance" "i-01a3c8837f5c1a2ab_Rahul_UF3":
  38:   vpc_security_group_ids = ["${data.terraform_remote_state.sg.aws_security_group_launch-wizard-9_sg-064b9d971a0df7c67_id}"]

This object has no argument, nested block, or exported attribute named
"aws_security_group_launch-wizard-9_sg-064b9d971a0df7c67_id".


Error: Unsupported attribute

  on instance.tf line 70, in resource "aws_instance" "i-01b4168f8073e7154_Rahul_HF2":
  70:   subnet_id         = "${data.terraform_remote_state.subnet.aws_subnet_subnet-5e33fa70_id}"

This object has no argument, nested block, or exported attribute named
"aws_subnet_subnet-5e33fa70_id".

```

**Comment by sergeylanzman** [comment]
ok, 
terraformer create code for terraform 0.11, not for 0.12
please run terraform 0.12upgrade
https://www.terraform.io/upgrade-guides/0-12.html

**Comment by clebio** [comment]
Does terraformer not support Classic (non-VPC) security groups? That seems to be the case, but I don't see it mentioned in the readme.

**Comment by sergeylanzman** [comment]
@clebio non-VPC  security groups it's very legacy. I see here in code https://github.com/GoogleCloudPlatform/terraformer/blob/master/providers/aws/sg.go#L38 non VPC skipped, but I think it's can be work, only not tested. It's first terraformer lines.....

**Comment by sergeylanzman** [comment]
I think issue resolved in master branch after add support 0.12
https://github.com/GoogleCloudPlatform/terraformer/pull/181
